Remove suggestion that order of table elements isn't important (#32909)
mdn/content: files/en-us/learn/html/tables/advanced/index.md

To use them:

- The `<thead>` element must wrap the part of the table that is the header — this is usually the first row containing the column headings, but this is not necessarily always the case. If you are using {{htmlelement("col")}}/{{htmlelement("colgroup")}} element, the table header should come just below those.

> - The `<tfoot>` element needs to wrap the part of the table that is the footer — this might be a final row with items in the previous rows summed, for example. The table footer should be the last element inside the table, if you have one.

Reviewer: I found "if you have one" slightly jarring and confusing. How about:

```suggestion
- The `<tfoot>` element needs to wrap the part of the table that is the footer — this might be a final row with items in the previous rows summed, for example. If included, the table footer should be the last element inside the table.
```

Author: Thanks @chrisdavidmills 

This is a good suggestion, I'm happy to accept it.

> - The `<tbody>` element needs to wrap the other parts of the table content that aren't in the table header or footer.

Reviewer 2: Would it be better to document `<thead>`, `<tbody>`, `<tfoot>` in that order, since that's now being prescribed as the order in which they should appear in the source? The copy for `<tbody>` would probably have to be reworded some, since it's not the last item on the list and can't rely on simply, "...and this is everything else", but I think it would still make the documentation flow more logically if the elements were documented in the same order they should be used.

Author: Thanks for the suggestion. This is a good idea - I had originally focused on just removing the problematic claim, but I don't mind making some more involved changes. I'll add a commit here which does that.
